ui/app/styles/components/http-requests-table.scss

```diff
@@ -0,0 +1,25 @@
+.http-requests-table {
+  & .is-collapsed {
+    visibility: collapse;
```

Comment:
this is needed to hide the table caption in the UI while making sure  that screen readers can still interpret it.